Source/plrmsg.cpp

```diff
@@ -81,9 +81,9 @@ void InitPlrMsg()
 void DrawPlrMsg()
 {
 	int i;
-	int x = 74;
-	int y = 230;
-	int width = 620;
+	DWORD x = 74;

```

Comment:
DrawPlrMsg is still bin exact with this change?

Reply:
Yes :)